configs/default/FLWO-FLYWOOF411EVO_HD.config

```diff
@@ -0,0 +1,100 @@
+# Betaflight / STM32F411 (S411) 4.1.1 Nov 15 2019 / 12:55:17 (1e5e3d369) MSP API: 1.42
+
+board_name FLYWOOF411EVO_HD
+manufacturer_id FLWO
+
+# resources
+resource BEEPER 1 C14
+resource MOTOR 1 A08
+resource MOTOR 2 A09
+resource MOTOR 3 A10
+resource MOTOR 4 B10
+resource PPM 1 A02
+resource PWM 1 B01
+resource LED_STRIP 1 B05
+resource SERIAL_TX 1 B06
+resource SERIAL_TX 2 A02
+resource SERIAL_RX 1 B07
+resource SERIAL_RX 2 A03
+resource I2C_SCL 1 B08
+resource I2C_SDA 1 B09
+resource LED 1 C13
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource ADC_BATT 1 A00
+resource ADC_RSSI 1 B01
+resource ADC_CURR 1 A01
+resource PINIO 1 B05
+resource FLASH_CS 1 B02
+resource OSD_CS 1 B12
```

Comment:
Does this board have a MAX7456 OSD chip?

If so `feature OSD` should be set.
If not, this setting is not needed and should be removed.

Please read and follow https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md